packages/record-data/types/@ember/debug/index.d.ts

```diff
@@ -0,0 +1,2 @@
+export { assert, warn, inspect } from '@ember/debug';
```

Comment:
Why do we need these specializations? The fact that this is an export from indicates that the upstream definitely typed types already have them. Looks like the only one it doesn't have is `deprecate`? Can you cross link an issue on definitelytyped to get this fixed? we shouldn't need to define this at all.

Reply:
there already is an open issue, and the way they were typed you can't just add deprecate.

Reply:
Ya, seems good can we link to that from a comment in here? Seems fine to make an issue with a checklist of all our overrides so we can audit them (and either remove them or add links to the issue)

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/38682